plugins/TopPairElectronPlusJetsSelectionFilter.cc

```diff
@@ -324,9 +324,13 @@ bool TopPairElectronPlusJetsSelectionFilter::passesLooseElectronVeto() const {
 			}
 		}
 		return isZEvent == true;
-	} else
-		// Require only one loose electron, which is the signal electron
+	} 
+	else {
+		if (nonIsolatedElectronSelection_ || invertedConversionSelection_){
```

Comment:
So to generate the noniso CR we invert a cut in the tight ID. We do not invert however the ID in the veto electron. We end up with one tight nonisolated electron from qcd and one veto isolated electron from somewhere else - mainly DY. The tight nonisolated no longer has a veto electron here.